index.html

```diff
@@ -611,7 +611,7 @@ <h2 id="steps">Steps for
 <a href="#trans-annc">transition announcement</a>
 to w3c-ac-members@w3.org
 <span data-profile="CR|PR|PR-OBSL|PR-RSCND|REC|OBSL|RSCND" data-cr="new|substantive|rec-update|rec-amended">and chairs@w3.org</span>
-<span data-profile="CR|PR|REC" data-cr="new|substantive|rec-update|rec-amended">and on the <a href="https://www.w3.org/">W3C home page</a></span>.
+<span data-profile="FPWD|CR|PR|REC" data-cr="new|substantive|rec-update|rec-amended">and posts a homepage news on the <a href="https://www.w3.org/">W3C home page</a></span>.
```

Comment:
This won't be reached because we don't have a transition announcement for FPWD. Do you just want a homepage news item but no email to w3c-ac-members of would you like both? If both, line 609 needs to include FPWD as well. If only the homepage news item, I'll propose a different update.

Reply:
You're correct that we don't have a trans-ann for FPWD, doh!
I just want a homepage news item.